client/src/redux/reducers/stage/service.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+import { API_URL } from '../../../constants/API_URL'
+
+export const fetchStage = async (id) => {
+  return await axios.get(`${API_URL}/stages/${id}`)
+}
+
+export const fetchStagesByTripId = async (tripId) => {
+  return await axios.get(`${API_URL}/stages`, { tripId: tripId })
+}
```

Comment:
with axios you cant pass tripId like this. need to use their config:
 ``` return await axios.get(`${API_URL}/stages`, { params: { tripId: tripId } })```
 or 
 ``` return await axios.get(`${API_URL}/stages`, { params: { tripId } })```
https://axios-http.com/docs/req_config

Reply:
ohhh no wonder i was getting a warning. fixed, thanks!